data_protection_base/__openerp__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': 'Data Protection Base',
+    'version': '9.0.1.0.0',
+    'category': '??',
+    'summary': 'Provides security and menus for data protection modules.',
+    'author': "Eficent, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.github.com/Eficent/gdpr,',
+    'license': 'AGPL-3',
+    'data': [
+        'security/data_protection.xml',
+        'views/data_protection_menu_view.xml',
+    ],
+    'installable': True,
```

Comment:
Add "application: True". This should the the App where further features will be anchored.